simespresso/io_QE/tests/test_start_espresso.py

```diff
@@ -0,0 +1,24 @@
+
+import unittest
+import os
+from simespresso.io_QE import espresso_class
+
+
+class OutcomesTest(unittest.TestCase):
+
+    def test_start_qe(self):
+        name_in = './pwtest.in'
+        name_out = './pwtest.out'
+        mpi=False
+        mpi_Nprocessors=2
+        path_to_espresso = '/usr/bin/pw.x'
+        print('qe wrapper attempting to run espresso')
+        wrapper = espresso_class.qe_functions()
+        wrapper.start_qe(name_in,name_out,path_to_espresso=path_to_espresso,mpi=mpi,mpi_Nprocessors=mpi_Nprocessors)
```

Comment:
This seems to fail on travis (https://travis-ci.org/simphony/simphony-espresso-dft/jobs/84774117#L446) because we don't have QE installed there yet.